submission-schema/build_schema.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
```

Comment:
I'd prefer to just drop this (we need just one method, to get us to the point where we'll start validating against the schema).